modules/globebrowsing/globebrowsingmodule.cpp

```diff
@@ -36,19 +38,69 @@
 #include <modules/globebrowsing/tile/tileprovider/tileproviderbylevel.h>
 #include <modules/globebrowsing/tile/tileprovider/tileproviderbyindex.h>
 
+#include <openspace/engine/openspaceengine.h>
 #include <openspace/rendering/renderable.h>
 #include <openspace/util/factorymanager.h>
 
 #include <ghoul/misc/templatefactory.h>
 #include <ghoul/misc/assert.h>
 
+#include <ghoul/systemcapabilities/generalcapabilitiescomponent.h>
+
 namespace openspace {
 
-GlobeBrowsingModule::GlobeBrowsingModule() : OpenSpaceModule("GlobeBrowsing") {}
+GlobeBrowsingModule::GlobeBrowsingModule()
+: OpenSpaceModule("GlobeBrowsing")
+{
+}
 
 void GlobeBrowsingModule::internalInitialize() {
     using namespace globebrowsing;
 
+    OsEng.registerModuleCallback(OpenSpaceEngine::CallbackOption::Initialize, [&]{
+        _openSpaceMaximumTileCacheSize = std::make_unique<properties::IntProperty>(
```

Comment:
I wanted to set the max value at creation but you are right. It is cleaner not having to use pointers.